build.assets/robotest/utils.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function branch {
+  # GRAVITY_GIT_VERSION_BRANCH_PREFIX may defined to accommodate remote names other than"origin"
```

Comment:
Nit: "may _be_ defined"? Also, space b/w "than" and "origin".

Reply:
"may" is intentional.  If `GRAVITY_GIT_VERSION_BRANCH_PREFIX` is undefined, `branch_prefix` defaults to `remotes/origin/version` which should be sane for CI and most developer setups.

This variable is to accommodate folks like me, who don't name their remotes 'origin':

```
walt@work:~/git/gravity$ git remote -v
grav    git@github.com:gravitational/gravity.git (fetch)
grav    git@github.com:gravitational/gravity.git (push)
walt    git@github.com:wadells/gravity.git (fetch)
walt    git@github.com:wadells/gravity.git (push)
```

Reply:
Whoops, this was a comment about the missing "be".  Will fix.

Reply:
Fixed in ec2a27a.